src/main/java/uk/gov/hmcts/reform/civil/helpers/bundle/BundleRequestMapper.java

```diff
@@ -420,6 +420,7 @@ private List<Element<BundlingRequestDocument>> mapStatementOfcaseDocs(CaseData c
         bundlingRequestDocuments.addAll(mapParticularsOfClaimDocs(caseData));
         List<Element<CaseDocument>> clAndDfDocList = caseData.getDefendantResponseDocuments();
         clAndDfDocList.addAll(caseData.getClaimantResponseDocuments());
+        clAndDfDocList.addAll(caseData.getSystemGeneratedCaseDocuments());
```

Comment:
is claimant response doc not part of system generated doc? will it not make it duplicate